templates/projects/tasks_browse.webpack.ejs

```diff
@@ -154,7 +154,7 @@
         <thead>
             <tr>
                 {% if 'task_id' in filter_data.display_columns %}
-                <th class="{{'sort-asc' if filter_data.order_by['task_id'] == 'asc'}} {{'sort-desc' if filter_data.order_by['task_id'] == 'desc'}} sortable" data-sort="task_id">
+                <th class="{{'sort-asc' if filter_data.order_by|length == 0 or filter_data.order_by['task_id'] == 'asc'}} {{'sort-desc' if filter_data.order_by['task_id'] == 'desc'}} sortable" data-sort="task_id">
```

Comment:
Could you explain a little bit the logic behind ```filter_data.order_by|length == 0```?  Looks like a bitwise OR operator of ```filter_data.order_by``` and ```length == 0```. Why it is used?

BTW, looks like your local branch is not sync'ed with "master" branch(There are some additional commits there).

Reply:
It does indeed have similar syntax to a bitwise OR. However, this is actually Jinja syntax for the [length operator](https://stackoverflow.com/a/1466550). :)

Regarding the commit history, this is a PR from the same branch of continued work for RDISCROWD-5067. The branch was merged with the main python3.9 and new commits added to continue the work. Thanks!